assets/unstructured/template/docker-build.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+# This script builds the container as per the `Dockerfile` in the same directory. Usually, only the
+# `container_name` variable should be adjusted to match the `container_id` in `config.yaml`
```

Comment:
Instead of requiring that the user updates the container name in two files, why not create a `container-name` file and ask them to update that? Then in this script and `docker-run.sh`, you can just do:

`container_name=$(cat "$script_dir/container-name")`